### PR TITLE
Log proper backoff messages on git clone

### PR DESCRIPTION
**Pull request by athos-ribeiro**

### CODE REVIEW

File: osbs/utils.py [hunk 1]
```diff
@@ -205,10 +205,14 @@ def subprocess_check_output_with_retry(args, max_retries=0, backoff_factor=0):
         except subprocess.CalledProcessError as ex:
             if counter < max_retries:
                 backoff = backoff_factor * (2 ** counter)
-                if backoff > 0:
-                    logger.info("retrying command '%s' in backoff seconds:\n '%s'", args, ex.output)
-                else:
+                if backoff <= 0:
                     logger.info("retrying command '%s':\n '%s'", args, ex.output)
+                elif backoff == 1:
```
Comment (twaugh) [inline osbs/utils.py]: Can avoid this test by wording it as "in {}s".

**Comment by MartinBasti** [comment]
Can we drop completely logic about handling wording second/seconds?  I think that everybody can handle to see `1 seconds` in a log (or use `second(s)`)

File: osbs/utils.py [hunk 2]
```diff
@@ -206,7 +206,8 @@ def subprocess_check_output_with_retry(args, max_retries=0, backoff_factor=0):
             if counter < max_retries:
                 backoff = backoff_factor * (2 ** counter)
                 if backoff > 0:
-                    logger.info("retrying command '%s' in backoff seconds:\n '%s'", args, ex.output)
+                    logger.info("retrying command '%s' in %s second(s):\n '%s'",
```
Comment (twaugh) [inline osbs/utils.py]: This looks awkward to me. Why not just the simpler `"... in %ss:\n '%s'"`? "1s", "30s" etc are not even a compromise, they are just how those durations are written.

**Comment by athos-ribeiro** [comment]
No release notes added. This was a fix for a bug introduced in this release cycle.

**Pull request merged**